### PR TITLE
All claims retirement content mg

### PR DESCRIPTION
**Pull request by markgreenburg**
## Description
- Updates military retired pay question / answer content
- Updates 'waive retirement pay' description / question / answer content

## Testing done
- Tested locally

## Screenshots
Military retired pay questionnaire:
screen shot 2018-11-13 at 12 46 51 pm

-----
Waive Retirement Pay page:
screen shot 2018-11-13 at 12 53 24 pm

## Acceptance criteria
- [x] Content updated per latest copy in ticket

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by peggygannon — CHANGES_REQUESTED** [review]
thanks, @markgreenburg Please see below for some additional tweaks: 

for table, can we swap the order of columns and fix the headers to have them match the guidance in the copy (see below)? 
Also, I think we're missing the strike-through in the table. 

 | Retirement            | Compensation           | Take home   |
| :-------------------- | :--------------------- | :---------- |
| $700/mo (minus taxes) | ~~$700/mo~~            | $630/mo     |
| **~~$700/mo~~**       | **$700/mo (tax-free)** | **$700/mo** |

2.  The footnote below the table should be styled as italics, no bold. 

cc @goldenmeanie for his fyi on the retirement table

**Comment by goldenmeanie** [comment]
Agree about the strikethrough, but I looking at it this way, I think the comp column first makes it read a little better. It puts the minus taxes right next to the lowered amount. I think this will be more clear.

File: src/applications/disability-benefits/all-claims/content/waiveRetirementPay.jsx [hunk 1]
```diff
@@ -1,14 +1,50 @@
...
+      <thead>
+        <tr>
+          <th>VA Compensation</th>
+          <th>Military Retirement</th>
+          <th>Take home pay</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>$700/month</td>
```
Comment (cvalarida) [inline src/applications/disability-benefits/all-claims/content/waiveRetirementPay.jsx]: Looks like a case for the [`<del>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/del)!

**Review by cvalarida — APPROVED** [review]
When it's good by @peggygannon and @goldenmeanie, it's good by me. :+1:

**Comment by markgreenburg** [comment]
@peggygannon @goldenmeanie thanks for the review, I was a little unsure of some of the specifics. How's this? Do the header names need to change?

screen shot 2018-11-19 at 10 58 25 am

**Comment by goldenmeanie** [comment]
Is it practical to force the parenthetical to the second line? I vote for this column order.

I am wondering if the strikethrough is legible enough? Is there a practical way to do something like this?
screen shot 2018-11-19 at 12 16 21 pm

**Comment by peggygannon** [comment]
@markgreenburg @goldenmeanie Hi, there. 

per, my latest edit, can we make the columns headers the below? 
 | Compensation           | Retirement          | Take home  pay |

**Comment by markgreenburg** [comment]
@goldenmeanie would it help to add row headers? E.g., something like:

|               | VA Compensation | Military Retired Pay |
|---------------|-----------------|----------------------|
| Compensation  | $700/month      | $700/month           |
| Taxes         | $0/month        | $70/month            |
| **Take Home Pay** | **$700/month**      | **$630/month**           |

**Comment by markgreenburg** [comment]
@peggygannon updated the headers:
screen shot 2018-11-19 at 12 53 09 pm

**Review by peggygannon — APPROVED** [review]

**Pull request merged**